src/app/team/shared/team-stage-select/team-stage-select.component.html

```diff
@@ -45,6 +45,9 @@
                </option>
             </select>
          </div>
+         <div *ngIf="competitionsBySeasonId[seasonIdFormValue]?.length === 0" class="d-flex align-items-center ml-2 mt-lg-2">
```

Comment:
Left space looks misaligned on small screens:
<img width="411" alt="Screenshot 2024-10-05 at 17 16 55" src="https://github.com/user-attachments/assets/66514c62-3732-41f9-9166-af00dbb618b3">

I suggest to use `col-*` classes:
```suggestion
         <div *ngIf="competitionsBySeasonId[seasonIdFormValue]?.length === 0" class="col-12 col-md-6 col-lg-4 col-xl-3 mt-lg-2">
```